README.md

```diff
@@ -0,0 +1,24 @@
+# postgres-to-redshift
+
+`postgres-to-redshift` copies postgres data to redshift via S3.
+
+## Running
+
+```bash
+AWS_REGION='us-east-1' \
+godep go run main.go \
+-redshifthost=<host> \
+-redshiftport=<port> \
+-redshiftuser=<user> \
+-redshiftpassword=<password> \
+-redshiftdatabase=<database> \
+-postgreshost=<host> \
+-postgresdatabase=<database> \
+-postgresuser=<user> \
+-postgresport=<port> \
+-postgrespassword=<password> \
+-s3prefix=<prefix> \
+-tables=<tables_csv>
+```
+
+In production, the binary is run on gearman using a standalone worker run as a cron job.
```

Comment:
Might acknowledge the datasci bash scripts that you first learned from -- you might also reach out to them in case they want to use this!